### PR TITLE
Feature/functional-pages-text

### PR DESCRIPTION
**Pull request by PotSchmoell**

### CODE REVIEW

**Review by SandroGlueck — APPROVED** [review]

**Comment by PotSchmoell** [comment]
@SandroGlueck I fixed a few issues in this PR. Can you have a quick look at those before we merge this one?
Sorry, I missed that @Th3da had already approved - If I would have realized that, I would have made a separate PR for the recent changes.

File: src/app/core/components/community-guidelines/community-guidelines.component.scss
Comment (SandroGlueck) [inline src/app/core/components/community-guidelines/community-guidelines.component.scss:9]: Did you test those styles on mobile aswell? 
Seems like there are missing media queries.

File: src/app/my-profile/components/my-profile/my-profile.component.html
Comment (SandroGlueck) [inline src/app/my-profile/components/my-profile/my-profile.component.html:20]: Is that necessary?

File: src/app/services/rating.service.ts
Comment (SandroGlueck) [inline src/app/services/rating.service.ts:36]: Thanks :D

File: src/app/review-item/components/review-success-page/review-success-page.component.html
Comment (SandroGlueck) [inline src/app/review-item/components/review-success-page/review-success-page.component.html:52]: Seems like a weird line break to me

**Review by SandroGlueck — CHANGES_REQUESTED** [review]

File: src/app/my-profile/components/my-profile/my-profile.component.html
Comment (PotSchmoell) [reply src/app/my-profile/components/my-profile/my-profile.component.html:20]: Of course not, I removed it.

File: src/app/review-item/components/review-success-page/review-success-page.component.html
Comment (PotSchmoell) [reply src/app/review-item/components/review-success-page/review-success-page.component.html:52]: You're right. Looks like autoformat didn't work too well on this one. I removed the linebreak

File: src/app/core/components/community-guidelines/community-guidelines.component.scss
Comment (PotSchmoell) [reply src/app/core/components/community-guidelines/community-guidelines.component.scss:9]: You're right. Fixed it and added a little padding on mobile.

**Review by SandroGlueck — APPROVED** [review]

**Pull request merged**
